pkgs/tools/graphics/spirv-cross/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, cmake, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "spirv-cross";
+  version = "2020-02-26";
+
+  src = fetchFromGitHub {
+    owner = "KhronosGroup";
+    repo = "SPIRV-Cross";
+    rev = "${version}";
+    sha256 = "0n9v8ynpwwkwcqzsb7qjl8nfy08pgmb9mn9w5hm9l1pxw6bfg1r0";
+  };
+
+  nativeBuildInputs = [ cmake python3 ];
+
+  meta = with stdenv.lib; {
+    description = "SPIRV-Cross is a practical tool and library for performing reflection on SPIR-V and disassembling SPIR-V back to high level languages";
```

Comment:
See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes. Also it may be worth considering the shorter first sentence in the README instead.